vms/platformvm/state/metadata_validator.go

```diff
@@ -32,6 +32,7 @@ type validatorMetadata struct {
 	LastUpdated              uint64        `v0:"true"` // Unix time in seconds
 	PotentialReward          uint64        `v0:"true"`
 	PotentialDelegateeReward uint64        `v0:"true"`
+	StakerStartTime          int64         `          v1:"true"`
```

Comment:
why is this int64?

Reply:
it's the type returned by time.Unix() method

Reply:
Should we do `uint64` to match `LastUpdated`?

Reply:
I don't think it is necessary. I can't recall why `LastUpdated` was typed as `uint64` in the first place. Currently we cast it to and from `int64` in order to convert it to a `time.Time`. I think we can avoid the type cast for `StakerStartTime`

Reply:
For better or worse we currently serialize all timestamps as `uint64`... We could change that in the future (`int64` and `uint64` have the same format). But I feel like for now we should use `uint64` just to align with the other usages.

Reply:
Changed to uint64. Happy to clean this up later on for all timestamps